debug/org.eclipse.debug.ui/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.debug.ui; singleton:=true
-Bundle-Version: 3.18.300.qualifier
+Bundle-Version: 3.18.400.qualifier
```

Comment:
> FYI, I would not trust the IDE's baseline information if you haven't deleted it and built it new:

Just in case: I'm not using Oomph for SDK development, I never update anything, I start everyday with latest nightly IBuild (if it was built at all).

Reply:
Sorry the comment was intended for @laeubi